src/components/Requirements.vue

```diff
@@ -1,149 +1,163 @@
 <template v-if="semesters">
+  <div class= "section">
   <div class="requirements">
-    <div class="fixed">
+    <div class="fixed" :class="{ bottomPreview: isBottomPreview && !isBottomBar, bottomBar: isBottomBar }">
```

Comment:
There seems to be an error where isBottomPreview and isBottomBar is not defined. Any reason these classes were added?

Reply:
This was here before Theresa changes so I wasn't sure what to do with it lol but it doesn't break if deleted so I just did.